easybuild/easyblocks/c/clang.py

```diff
@@ -491,22 +514,48 @@ def sanity_check_step(self):
         if LooseVersion(self.version) >= LooseVersion('3.8'):
             custom_paths['files'].extend(["lib/libomp.%s" % shlib_ext, "lib/clang/%s/include/omp.h" % self.version])
 
+        # Detect OpenMP support for CPU architecture
+        arch = get_cpu_architecture()
+        # Check architecture explicitly since Clang uses potentially
+        # different names
+        if arch == X86_64:
+            arch = 'x86_64'
+        elif arch == POWER:
+            arch = 'ppc64'
+        elif arch == AARCH64:
+            arch = 'aarch64'
+        else:
+            print_warning("Unknown CPU architecture (%s) for OpenMP library check!" % arch)
+        custom_paths['files'].extend(["lib/libomptarget.%s" % shlib_ext,
+                                      "lib/libomptarget.rtl.%s.%s" % (arch, shlib_ext)])
+        # If building for CUDA check that OpenMP target library was created
         if 'NVPTX' in self.cfg['build_targets']:
-            arch = get_cpu_architecture()
-            # Check architecture explicitly since Clang uses potentially
-            # different names
-            if arch == X86_64:
-                arch = 'x86_64'
-            elif arch == POWER:
-                arch = 'ppc64'
-            elif arch == AARCH64:
-                arch = 'aarch64'
-            else:
-                print_warning("Unknown CPU architecture (%s) for OpenMP offloading!" % arch)
-            custom_paths['files'].extend(["lib/libomptarget.%s" % shlib_ext,
-                                          "lib/libomptarget-nvptx.a",
```

Comment:
@nordmoen Are you intentionally no longer checking for `lib/libomptarget-nvptx.a` (you're only adding `lib/libomptarget.rtl.cuda.*` below)

Reply:
That was a mistake on my part, fixed now